src/locales.json

```diff
@@ -17,7 +17,7 @@
         "search": "Suche",
         "events": "News",
         "extras": "Extras",
-        "landing": "Wo bist du?",
+        "landing": "Ortsauswahl",
```

Comment:
The issue descrption was a bit unclear. Please also change this in the other languages. For languages like farsi or arabic you can omit the key currently.

Reply:
If you omit a key it will fallback automatically to english